win32ss/user/ntuser/sysparams.c

```diff
@@ -253,7 +253,7 @@ SpiUpdatePerUserSystemParameters(VOID)
     gspv.ncm.iCaptionWidth = SpiLoadMetric(L"CaptionWidth", 19);
     gspv.ncm.iCaptionHeight = SpiLoadMetric(L"CaptionHeight", 19);
     gspv.ncm.iSmCaptionWidth = SpiLoadMetric(L"SmCaptionWidth", 12);
-    gspv.ncm.iSmCaptionHeight =  SpiLoadMetric(L"SmCaptionHeight", 14);
+    gspv.ncm.iSmCaptionHeight =  SpiLoadMetric(L"SmCaptionHeight", 15);
```

Comment:
Since you're at it could you remove the extra space that is before "SpiLoadMetric"? Thkx

Reply:
OK.